samples/emoji-search/presenter/src/commonMain/kotlin/com/example/redwood/emojisearch/presenter/EmojiSearch.kt

```diff
@@ -123,9 +123,18 @@ private fun LazyColumn(
   }
 
   val lazyListState = rememberLazyListState()
+//
+//  LaunchedEffect(searchTerm) {
+//    println("**** all emojis size: ${allEmojis.size}")
+//    lazyListState.scrollToItem(lazyListState.firstVisibleItemIndex)
+//  }
 
-  LaunchedEffect(searchTerm) {
-    lazyListState.scrollToItem(0)
+  LaunchedEffect(allEmojis.size) {
+    if (allEmojis.size > lazyListState.firstVisibleItemIndex) {
+      println("**** all emojis size: ${allEmojis.size}")
+      println("**** first visible item index: ${lazyListState.firstVisibleItemIndex}")
+      lazyListState.scrollToItem(lazyListState.firstVisibleItemIndex)
```

Comment:
log

output of logs for the following events
1) scroll down on the list
2) click on an emoji, UI will navigate away to chrome 
3) click back, UI will navigate back to app 

<img width="1041" alt="Screenshot 2023-10-12 at 10 38 36 AM" src="https://github.com/cashapp/redwood/assets/57155915/577804ed-29b5-4b18-9a9c-4407e2fe54a1">

but list stays on top (doesn't scroll to position)